### DISCUSSION TITLE
please add start and end instead left and right

### DISCUSSION THREAD

**Discussion by nekooee** [tailwindlabs/tailwindcss#3930] [Ideas]
Hi,
I current used this package for multi-language:
https://github.com/20lives/tailwindcss-rtl

It's worked fine, but not supported add style in live (example: ms-[20px] not work). If you add the start and end to the main package then we can use all the features easily.

**Comment by mmaseraj** [comment]
It would be awesome if this can be implemented by checking the [dir] in the the root <html> or <body>, because now working in RTL is messy and not ideal. I suggest adding this as an option in the config file.
